webcalc/templates/about.html

```diff
@@ -1,9 +1,153 @@
+                </p>
+                <p>
+                    This metric reflects the probability of words under a simple bigram model. The probability of a word is the product of the probability of all the bigrams in contains. Note that the probability of the bigrams is based only on their frequency of occurrence, not the position in which they occur or their sequencing with respect to one another.
+                </p>
+            </li>
+            <li>
+                <p>
+                    <b>Positional unigram score</b> (<tt>pos_uni_prob</tt>): This is a type-weighted variant of unigram score from Vitevich and Luce (2004). 
+
+                    {% latexify 'PosUniScore(w=x_1 \dots x_n) \approx 1 + \sum_{i=1}^{n} P(w_i = x_i)' math_block=True %}
+
+                    where
+
+                    {% latexify 'P(w_i = x) = \frac{C(w_i = x)}{\displaystyle\sum_{y \in \Sigma} C(w_i = y)}' math_block=True %}
+
+                    where {% latexify 'w_i' math_inline=True %} refers to the {% latexify 'i^{\text{th}}' math_inline=True %} position in a word and {% latexify 'C(w_i = x)' math_inline=True %} is the number of times in the training data the symbol {% latexify 'x' math_inline=True %} occurs in the {% latexify 'i^{\text{th}}' math_inline=True %} position of a word.
+                </p>
+                <p>
+                    Under this metric, the score assigned to a word is based on the sum of the probability of its individual symbols occuring at their respective positions. Note that the ordering of the symbols with respect to one another does not affect the score, only their relative frequencies within their given positions. Higher scores represent words with more probable phonotactics, but note that this scores cannot be interpreted as probabilities.
+                </p>
+            </li>
+            <li>
+                <p>
+                    <b>Positional bigram score</b> (<tt>pos_bi_prob</tt>): This is a type-weighted variant of the bigram score from Vitevich and Luce (2004). 
+
+                    {% latexify 'PosBiScore(w=x_1 \dots x_n) \approx 1 + \sum_{i=2}^{n} P(w_{i-1} = x_{i-1}, w_i = x_i)' math_block=True %}
+
+                    where
+
+                    {% latexify 'P(w_{i-1} = y, w_i = x) = \frac{C(w_{i-1} = y, w_i = x)}{\displaystyle\sum_{z \in \Sigma}\sum_{v \in \Sigma} C(w_{i-1} = z, w_i = v)}' math_block=True %}
+
+                    where {% latexify 'w_i' math_inline=True %} refers to the {% latexify 'i^{\text{th}}' math_inline=True %} position in a word and {% latexify 'C(w_{i-1} = y, w_i = x)' math_inline=True %} is the number of times in the training data the sequence {% latexify 'yx' math_inline=True %} occurs at the {% latexify '(i-1)^{\text{th}}' math_inline=True %} and {% latexify 'i^{\text{th}}' math_inline=True %} positions of a word.
```

Comment:
Do we have to define $w_i$ again here even though we defined it for the positional unigram score?

Reply:
I don't think it hurts to redefine it here in case people are skipping around